libs/test/Test/Golden.idr

```diff
@@ -227,7 +234,8 @@ runTest opts testPath = forkIO $ do
       case str of
```

Comment:
```suggestion
      case toLower str of
```

If the prompt would be changed to `[y/N]` it might help avoid some casing confusion.

Reply:
I think it's standard to have upper case used to indicate the default.

Reply:
Yeah, but it is more of a "ignore casing just in case".